apps/ewallet_db/lib/ewallet_db/auth_token.ex

```diff
@@ -203,6 +231,7 @@ defmodule EWalletDB.AuthToken do
     })
   end
 
+  @spec expire_for_user(atom() | User.t()) :: :ok
```

Comment:
```suggestion
  @spec expire_for_user(atom() | %User{}) :: :ok
```

We don't have `User.t()` for now